README.md

```diff
@@ -99,6 +99,13 @@ const doc = collection.doc('my-id');
 
 When a query has been created, you use the `useQuery` hook to execute it, and optionally `subscribe` to the results:
 
+The hook returns an object that includes the following properties:
+
+- `loading`: Whether data has been returned by the query.
+- `docs`: The query results as an array of document references.
+- `data`: The query results as an array of data (same as calling `docs.map(d => d.data)`).
+- `error`: The error object, if an error occurs while executing the query.
+
 ```jsx
 function App() {
   const collection = useCollection('my-collection');
```

Comment:
maybe make this a typescript example? (add types)

Reply:
Done. Once the collection is typed the types carry through everywhere else.